### PR TITLE
Don't skip network policies test for crc

### PR DESCRIPTION
**Pull request by Frawless**
Signed-off-by: Jakub Stejskal <xstejs24@gmail.com>

### Type of change

_Select the type of your PR_

- Bugfix
- Enhancement / new feature

### Description
This PR fix the issue with skipped `NetworkPolicy` tests on crc environment and add some more info to results sent from Jenkins.

### Checklist

- [x] Make sure all tests pass

### CODE REVIEW

**Comment by Frawless** [comment]
@strimzi-ci run tests crc profile=networkpolicies test_only

File: Jenkinsfile-pr [hunk 1]
```diff
@@ -23,7 +26,7 @@ pipeline {
     parameters {
         string(name: 'TEST_CASE', defaultValue: '*ST', description: 'maven parameter for executing specific tests')
         string(name: 'TEST_PROFILE', defaultValue: 'acceptance', description: 'maven parameter for executing specific test profile')
-        string(name: 'EXCLUDE', defaultValue: 'networkpolicies', description: 'maven parameter for exclude specific test tag')
+        string(name: 'EXCLUDE', defaultValue: ${DEFAULT_EXCLUDED_GROUPS}, description: 'maven parameter for exclude specific test tag')
```
Comment (im-konge) [inline Jenkinsfile-pr:29]: I think this will fail the build -> it should be like this: `"${DEFAULT_EXCLUDED_GROUPS}"` - or am I wrong?

**Comment by Frawless** [comment]
@strimzi-ci run tests crc profile=networkpolicies test_only

**Comment by im-konge** [comment]
The `testNetworkPoliciesWithTlsListener` and `testNetworkPoliciesWithPlainListener` will be fixed by me in #3024

**Comment by Frawless** [comment]
@strimzi-ci run tests crc profile=networkpolicies test_only

**Comment by Frawless** [comment]
@strimzi-ci run tests crc profile=networkpolicies test_only

**Comment by Frawless** [comment]
@strimzi-ci run tests crc profile=networkpolicies test_only

**Comment by Frawless** [comment]
@strimzi-ci run tests crc profile=networkpolicies test_only

**Comment by Frawless** [comment]
@strimzi-ci run tests profie=acceptance testcase=UserST

**Comment by Frawless** [comment]
@strimzi-ci run tests crc profile=networkpolicies test_only

**Comment by Frawless** [comment]
@strimzi-ci run tests profie=acceptance testcase=UserST test_only

**Pull request merged**